README.md

```diff
@@ -43,8 +43,21 @@ This project assumes that your security team has already created VPCs, security_
 - automate_route_table_id - The [Route Table](https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Route_Tables.html) under which all aws resources will be created.
 
 
+# Delivery validator keys
+
+You must provide your own Delivery validator key.
+
 # Builder keys
 
-You must provide a public/private builder key pair. They must be in ``.keys/builder_key` and ``.keys/builder_key.pub` for now.
+You no longer need to provide builder keys. Once you've generated a `delivery-validator.pem` using `ssh-keygen` or your other favorite key generator, you can generate the public key in the format needed by the Chef server by using a command like `openssl rsa -in delivery-validator.pem -pubout -out delivery-validator.pub`.
+
```

Comment:
I just added local exec steps in the tf plan to deal with this, why make people type :)